examples/email-auth-local-storage/README.md

```diff
@@ -65,3 +67,30 @@ $ pnpm run dev
 ```
 
 This command will run a NextJS app on port 3000. If you navigate to http://localhost:3000 in your browser, you can follow the prompts to start an email auth.
+
+### 4/ Stackblitz Example
+
+You will need to create a Stackblitz account to run the following example
+
+Example Link: https://stackblitz.com/edit/stackblitz-starters-pyyw59
+
+#### Prerequisites
+
+To use the example you need the following
+
+- Turnkey Organization
+- API key for the root user (both public and private key needed)
+- Turnkey Suborg with an email user
+
+#### Set Env Variables
+
+Set the following environment variables in the `.env.local` file
```

Comment:
We may want to set these variables in the stackblitz's `.env.local` and remove the values that you've included in there

Reply:
Yeah the stackblitz `.env.local` only has placeholders right now no values!

Reply:
Ah looks like I forgot email auth local storage

Reply:
Actually they seemed finnicky -- some that I changed earlier reloaded to previous values. 

Have confirmed that the placeholder values are in now